list_options.go

```diff
@@ -10,3 +10,19 @@ type ListOptions struct {
 	EndDateTime   *time.Time `url:"end_datetime,omitempty"`
 	PageSize      int        `url:"page_size,omitempty"`
 }
+
+// nextPageParameters returns the next page of a paginated collection.
+// If pageSize is 0, it will use the default page size
+func nextPageParameters(hasMore bool, lastId string, pageSize int) (out *ListOptions, err error) {
+	if !hasMore {
+		err = EndOfPaginationError
+		return
+	}
+	params := &ListOptions{
+		BeforeID: lastId,
+	}
+	if pageSize > 0 {
+		params.PageSize = pageSize
+	}
+	return params, err
```

Comment:
this method is using named return params. The first named param is `out`, but is never used